aspnet/fundamentals/dependency-injection.rst

```diff
@@ -113,35 +108,29 @@ Scoped
   Scoped lifetime services are created once per request.
 
 Singleton
-  Singleton lifetime services are created the first time they are requested, and then every subsequent request will use the same instance. If your application requires singleton behavior, allowing the services container to manage the service's lifetime is recommended instead of implementing the singleton design pattern and managing your object's lifetime in the class yourself.
-
-Instance
-  You can choose to add an instance directly to the services container. If you do so, this instance will be used for all subsequent requests (this technique will create a Singleton-scoped instance). One key difference between ``Instance`` services and ``Singleton`` services is that the ``Instance`` service is created in ``ConfigureServices``, while the ``Singleton`` service is lazy-loaded the first time it is requested.
+  Singleton lifetime services are created the first time they are requested (or when ``ConfigureServices`` is run if you specify an instance there) and then every subsequent request will use the same instance. If your application requires singleton behavior, allowing the services container to manage the service's lifetime is recommended instead of implementing the singleton design pattern and managing your object's lifetime in the class yourself.
 
 Services can be registered with the container in several ways. We have already seen how to register a service implementation with a given type by specifying the concrete type to use. In addition, a factory can be specified, which will then be used to create the instance on demand. The third approach is to directly specify the instance of the type to use, in which case the container will never attempt to create an instance.
 
 To demonstrate the difference between these four lifetime and registration options, consider a simple interface that represents one or more tasks as an *operation* with a unique identifier, ``OperationId``. Depending on how we configure the lifetime for this service, the container will provide either the same or different instances of the service to the requesting class. To make it clear which lifetime is being requested, we will create one type per lifetime option:
```

Comment:
You no longer have four. So just make it 
To demonstrate the difference between these lifetimes, ...